style/GenreSelectCss.elm

```diff
@@ -0,0 +1,228 @@
+module GenreSelectCss exposing (..)
```

Comment:
I took this file from the multiselect source examples, and just modified a few things. Usually, they'd export their example styles or at least expose their css classes.